ostree/ostree_dest.go

```diff
@@ -22,6 +22,11 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/ostreedev/ostree-go/pkg/otbuiltin"
+
```

Comment:
(Nit: Please keep the imports organized in two sections, imports from the default Golang library, and all external imports.)

Reply:
ping